### DISCUSSION TITLE
Control Q Drawer Animation Speed

### DISCUSSION THREAD

**Discussion by Code-Case** [quasarframework/quasar#8992] [General - Components / Directives / etc]
Hello,

I like to open the QDrawer list delayed after 1 second on mouseover, 
Is there an api/slot like delay animation: 1000 or something? 

Thank you

**Comment by ssuess** [comment]
I have a similar question, how can the speed of opening (I guess the duration of the animation) be controlled in q-drawer?

**Reply by ssuess** [reply]
Still wondering is there no supported way to do this without modifying the core code?

**Reply by metalsadman** [reply]
play around with the delay values of css used. 
```sass
.q-body--layout-animate
  .q-drawer__backdrop
    transition: background-color .12s !important
  .q-drawer
    transition: transform .12s, width .12s, top .12s, bottom .12s !important
  .q-layout__section--marginal
    transition: transform .12s, left .12s, right .12s !important
  .q-page-container
    transition: padding-top .12s, padding-right .12s, padding-bottom .12s, padding-left .12s !important
  .q-page-sticky
    transition: transform .12s, left .12s, right .12s, top .12s, bottom .12s !important
```

**Reply by ssuess** [reply]
at first I was trying playing with those values and saw no difference, but now I think I see I have a different problem, because I have an element that is contained by the drawer that has it's own timing that I was trying to match to the drawer but no luck. These do seem to affect the drawer itself, I will keep looking for ways to match the containing element transition, thanks.
